PerformanceTest_Static_PressureDorsal.py

```diff
@@ -80,13 +90,13 @@ def createAvgMat(inputName):
     """
    
         
-    inputName = entries[4]
-   
-    dat = pd.read_csv(fPath+inputName, sep=',', header = 'infer')
+    # inputName = entries[0]
+    # dat = pd.read_csv(fPath+inputName, sep=',',skiprows = 1, header = 'infer')
+    dat = pd.read_csv(fPath+inputName, sep=',',header = 'infer')
     subj = inputName.split(sep="_")[0]
     config = inputName.split(sep="_")[1].split(sep=".")[0]
     
-    insoleSide = dat['Insole Side'][0]
+    insoleSide = dat['Insole.1'][0] #Depending on the export, might be names "Insole Side"
```

Comment:
I have only seen 'Insole' or 'Insole.1' with this newer (May24) update of xsensor.